lookout/style/typos/tests/test_corrector_utils.py

```diff
@@ -1,21 +1,21 @@
-import unittest
+from os.path import join
 import pickle
+import unittest
 
-import pandas
 import numpy
+import pandas
 from pandas.util.testing import assert_frame_equal
 
-from lookout.style.typos.utils import (flatten_data, add_context_info, rank_candidates,
-                                       filter_suggestions, suggestions_to_df,
-                                       suggestions_to_flat_df)
+from lookout.style.typos.utils import (add_context_info, filter_suggestions, flatten_data,
+                                       rank_candidates, suggestions_to_df, suggestions_to_flat_df)
 
-TEST_DATA_PATH = "lookout/style/typos/tests"
+TEST_DATA_PATH = "lookout/style/typos/tests/"
```

Comment:
`TEST_DATA_PATH` should be defined as `str(pathlib.Path(__file__).parent)` so that the tests can be called from anywhere.